quickstarts/install-secrets-store-csi.yaml

```
@@ -0,0 +1,161 @@
+          topic.
+    - title: Deploy the ClusterCSIDriver custom resource
+      description: |-
+        ### What is ClusterCSIDriver?
+
+        The ClusterCSIDriver custom resource instructs the operator to deploy and manage the Secrets Store CSI driver
+        on your cluster. Once deployed, the CSI driver will run as a DaemonSet on all nodes, ready to mount secrets
+        from external sources into your pods.
+
+        ### To deploy the ClusterCSIDriver custom resource:
+
+        1. From the **Administrator** perspective, navigate to **Administration** → **CustomResourceDefinitions** from the [Administration]{{highlight qs-nav-administration}} section.
+
+        1. In the **Search by name** field, type `ClusterCSIDriver`.
+
+        1. Click on the **ClusterCSIDriver** custom resource definition from the list.
+
+        1. Click on the **Instances** tab.
+
+        1. Click **Create ClusterCSIDriver** button.
+
+        1. In the YAML editor, use the following configuration:
+
+        ```yaml
+        apiVersion: operator.openshift.io/v1
+        kind: ClusterCSIDriver
+        metadata:
+          name: secrets-store.csi.k8s.io
+        spec:
+          managementState: Managed
+        ```
+
+        1. Click **Create** to deploy the ClusterCSIDriver.
+
+        1. Wait for the CSI driver pods to be deployed across your cluster nodes.
+      review:
+        instructions: |-
+          #### Verify the ClusterCSIDriver was successfully deployed:
+
+          1. Navigate to **Workloads** → **DaemonSets** from the [Workloads]{{highlight qs-nav-workloads}} section.
+
+          1. In the **Project** dropdown menu at the top, select **openshift-cluster-csi-drivers**.
+
+          1. Look for **csi-secrets-store** and **csi-secrets-store-node** DaemonSets.
+
+          1. Verify that the desired number of pods matches the current number of pods for each DaemonSet.
+
+          Are all CSI driver pods running successfully?
+        failedTaskHelp: >-
+          This task isn't verified yet. Try the task again. Check the operator
+          logs if the CSI driver pods are not deploying correctly.
+      summary:
+        success: >-
+          You have successfully deployed the Secrets Store CSI Driver! You can
+          now create SecretProviderClass resources and mount secrets from external
+          sources as volumes in your pods.
+        failed: >-
+          This task is incomplete. Check your work to make sure the ClusterCSIDriver
+          was created correctly.
+  conclusion: >-
+    Your Secrets Store CSI Driver is ready! You can now create SecretProviderClass
+    resources to define connections to external secret management systems and mount
+    secrets as volumes in your workloads. To learn more, refer to the
+    [OpenShift documentation](https://docs.openshift.com/container-platform/latest/storage/container_storage_interface/persistent-storage-csi-secrets-store.html).
```

This uses a link to docs.openshift.com, which isn't in use anymore. It should redirect, but it would be a better experience for users if this went directly to docs.redhat.com. I'd also recommend going to the section in the nodes book, which has more of the actual steps of using the operator. The storage book only has about installing the operator.

```suggestion
    [OpenShift documentation](https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/nodes/working-with-pods#nodes-pods-secrets-store).
```